### DISCUSSION TITLE
Was Working....Now It's Not

### DISCUSSION THREAD

**Discussion by zeethreee** [MonitorControl/MonitorControl#1454] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

 checked Better Display and Lunar out, they’re nice, but I like monitor control best.

I think I might have made some kind of mess, though.

When I removed the others I went to change the contrast in Monitor control of my Alogic monitor and it flipped to grayscale. 
I rebooted and it was fine.

I removed MonitorControl and reinstalled it. It’s now not syncing between the two. The app launches and I can see both monitors, but the display brightness doesn’t pop up on the Alogic and it doesn’t change as well.
I tried to reset all settings - the same result.

Any suggestions?

### Expected behavior

normal functionality 

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.0
- Mac model: MBP 2021
- MonitorControl version: 4.2.0
- Monitor(s): Studio Display, Alogic 27"
- Monitor Cable(s)/Connection(s): USB-C, USB-c to HDMI
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
I'll move this to discussions, maybe there are others with experience in Alogic monitors.
